### PR TITLE
Add Up Xtreme i12 firmware update instruction

### PR DESCRIPTION
**Pull request by tsaikevin**
Add steps to create capsule and trigger firmware update.

### CODE REVIEW

File: source/supported-hardware/upxtremei12.rst [hunk 1]
```diff
@@ -90,6 +90,54 @@ Stitch |SPN| images with factory BIOS image using the stitch tool::
 
 See :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
 
+Slimbootloader binary for capsule
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A capsule image could ecapsulate a Slimbootloader image to be used in firmware update mechanism. More information is described in :ref:`firmware-update`.
+
+For the process running properly, an IFWI image programmed on board with TOP SWAP size configuration is required.
+
+.. note:: Enabling IFWI capsule update feature requires additional firmware components and tools.
```
Comment (rprangar) [inline source/supported-hardware/upxtremei12.rst:99]: Can you clarify what is referred to as IFWI capsule update ?

Comment (tsaikevin) [reply source/supported-hardware/upxtremei12.rst:99]: @rprangar here IFWI capsule update feature is referring to TOP SWAP size setting,  To be specific maybe just re-write as TOP SWAP size

Comment (tsaikevin) [reply source/supported-hardware/upxtremei12.rst]: @rprangar edited, pls check again.

**Review by rprangar — APPROVED** [review]
LGTM

**Pull request merged**